tests/integration/test_subscribe.py

```diff
@@ -200,15 +200,18 @@ def test_immediate_subscribe(self):
                               accept=lambda x: self.wlockExists(x, False))
         self.assertNotEqual(None, dat)
 
+    @unittest.skipIf(os.name == 'nt', 'win')
```

Comment:
You shouldn't skip the whole test here because most of it actually works on Windows. You should either drop the Unicode name on Windows or create a separate test (which can be simplified from the original `test_subscribe`.